frame/scored-pool/src/lib.rs

```diff
@@ -409,7 +422,7 @@ pub mod pallet {
 		#[pallet::weight(0)]
 		pub fn change_member_count(origin: OriginFor<T>, count: u32) -> DispatchResult {
 			ensure_root(origin)?;
-			MemberCount::<T, I>::put(&count);
+			Self::update_member_count(count).map_err(|_| Error::<T, I>::TooManyMembers)?;
```

Comment:
```suggestion
			Self::update_member_count(count)?;
```
Isnt this already returning the correct error type?